lxd/migrate_instance.go

```diff
@@ -446,26 +463,24 @@ func (s *migrationSourceWs) Do(state *state.State, migrateOp *operations.Operati
 	if s.instance.Type() == instancetype.VM {
 		blockSize, err := storagePools.InstanceDiskBlockSize(pool, s.instance, migrateOp)
 		if err != nil {
-			return fmt.Errorf("Failed getting source disk size: %w", err)
+			return abort(fmt.Errorf("Failed getting source disk size: %w", err))
 		}
 
-		logger.Debugf("Set migration offer volume size for %q: %d", s.instance.Name(), blockSize)
+		l.Debug("Set migration offer volume size for %q: %d", logger.Ctx{"blockSize": blockSize})
```

Comment:
`%q` and `%d` can be removed as they aren't used

Reply:
Good spot thanks!